docs/book/driver-deployment/installation.md

```diff
@@ -0,0 +1,226 @@
+
+- `NetPermissions` - optional parameter. Set of parameters used to restrict the network capabilities of all the file share volumes created under this vSphere configuration. If the complete set of `NetPermissions` are not mentioned for a given IP range, defaults are assumed for the missing parameters. You can define as many `NetPermissions` sections as you want and each of these sections is uniquely identified by the string which follows.
+
+The parameters grouped by `NetPermissions` are as follows:
+
+- `Ips` - defines the IP range or IP subnet to which these restrictions will be levied upon. The default value for `Ips` is "*" which means all the IPs.
+
+- `Permissions` - can either be "READ_WRITE", "READ_ONLY" or "NO_ACCESS". The default value for `Permissions` is "READ_WRITE" for the given IP range.
+
+- `RootSquash` - defines the security access level for the file share volume. The default for `RootSquash` is `false` i.e allow root access to the all the file share volumes created within the given IP range.
+
+## Create a kubernetes secret for vSphere credentials
+
+Create a Kubernetes secret that will contain configuration details to connect to vSphere.
+
+Create the secret by running the following command:
+
+```bash
+kubectl create secret generic vsphere-config-secret --from-file=csi-vsphere.conf --namespace=kube-system
+```
+
+Verify that the credential secret is successfully created in the kube-system namespace.
+
+```bash
+$ kubectl get secret vsphere-config-secret --namespace=kube-system
+NAME                    TYPE     DATA   AGE
+vsphere-config-secret   Opaque   1      43s
+```
+
+For security purposes, it is advised to remove this configuration file.
+
+```bash
+rm csi-vsphere.conf
+```
+
+## Create Roles, ServiceAccount and ClusterRoleBinding for vSphere CSI Driver
+
+Create ClusterRole, ServiceAccounts and ClusterRoleBinding needed for installation of vSphere CSI Driver
+
+```bash
+$ kubectl apply -f https://raw.githubusercontent.com/kubernetes-sigs/vsphere-csi-driver/master/manifests/1.14/rbac/vsphere-csi-controller-rbac.yaml
+serviceaccount/vsphere-csi-controller created
+clusterrole.rbac.authorization.k8s.io/vsphere-csi-controller-role created
+clusterrolebinding.rbac.authorization.k8s.io/vsphere-csi-controller-binding created
+```
+
+Click [here](https://github.com/kubernetes-sigs/vsphere-csi-driver/tree/master/manifests/vsphere-7.0/vanilla/rbac) to view the roles assigned to vSphere CSI Driver.
+
+## Install vSphere CSI driver
+
+Our CSI Controller runs as a Kubernetes deployment, with a replica count of 1. For version `v2.0.0`, the `vsphere-csi-controller` Pod consists of 6 containers – the CSI controller, External Provisioner, External Attacher, External Resizer, Liveness probe and vSphere Syncer. The vSphere Syncer introduces features supported by our [Cloud Native Storage solution](https://docs.vmware.com/en/VMware-vSphere/6.7/Cloud-Native-Storage/GUID-51D308C7-ECFE-4C04-AD56-64B6E00A6548.html)
```

Comment:
`The vSphere Syncer introduces features supported by our ` -> this is not clear

Let's link to 7.0 code.

Reply:
done